fe/src/main/java/org/apache/doris/catalog/TableProperty.java

```diff
@@ -0,0 +1,140 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package org.apache.doris.catalog;
+
+import com.google.common.base.Strings;
+import org.apache.doris.common.io.Text;
+import org.apache.doris.common.io.Writable;
+import org.apache.doris.common.util.PropertyAnalyzer;
+
+import java.io.DataInput;
+import java.io.DataOutput;
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+
+public class TableProperty implements Writable {
+    private Map<String, String> dynamicProperties = new HashMap<>();
+
+    public static final String LAST_UPDATE_TIME = "LastUpdateTime";
+    public static final String MSG = "Msg";
+    public static final String STATE = "State";
+
+    private long dbId;
+    private long tableId;
```

Comment:
Can't save these two ids.

Reply:
The reason to save these two ids is to use for  replayModifyTableDynamicPartition
`long dbId = tableProperty.getDbId();
        long tableId = tableProperty.getTableId();
        Map<String, String> dynamicProperties = tableProperty.getDynamicProperties();

        Database db = getDb(dbId);
        db.writeLock();
        try {
            OlapTable table = (OlapTable) db.getTable(tableId);
            table.setTableProperty(new TableProperty(dbId, tableId, dynamicProperties));
        } finally {
            db.writeUnlock();
        }`

Reply:
I don't think so. Property is used to give table's property.

Reply:
> I don't think so. Property is used to give table's property.

If can't save these two ids, is there another way to get tableId to do replayModifyTableDynamicPartition?